backend/src/routes/auth.js

```diff
@@ -85,53 +100,42 @@ track of this transaction
 to authenticate into other endpoints
 */
 async function handleLogin(req, res) {
-  const { idToken } = req.body;
```

Comment:
I think it is still necessary to pass a token to this handler. We want the admin API to verify the token. Otherwise, someone can initiate 2-factor auth for an account that isn't theirs via this endpoint just by providing an email in `req.locals.user`.

Reply:
Also wouldn't req.locals not even be set at this handler? Based on how the routes were configured it doesn't look like we go through any middleware before reaching this block.

Reply:
idToken is still used--but idToken should be passed via the login_token header, not via request body.

firebaseAuthMiddleware retrieves the user's e-mail using the idToken and stores it under req.locals.user.email, and this handler uses that value.